Docs for Microsoft Community Health (#41)
SteeltoeOSS/steeltoe-site: source/docs/steeltoe-connectors.html.markdown

        return healthContributors;
    }
```

## Using Microsoft Community HealthChecks for ASP.NET Core
ASP.NET Core also offers [Middleware and libraries](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/health-checks?view=aspnetcore-2.

> ) and abstractions for reporting health. There is wide community support for these abstractions from libraries such as [AspNetCore.Diagnostics.HealthChecks](https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks). It is now possible to use these community provided health checks and make them available via Steeltoe management health endpoint (for integration with PCF or any other infrastructure that depends on this format). In addition Steeltoe connectors now exposes functionality to easily get connection information which is needed to setup these Microsoft Health Checks. 

Reviewer: ```suggestion
) and abstractions for reporting health. There is wide community support for these abstractions from libraries such as [AspNetCore.Diagnostics.HealthChecks](https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks). It is now possible to use these community provided health checks with the Steeltoe management health endpoint (for integration with PCF or any other infrastructure that depends on this format). In addition, Steeltoe connectors now expose functionality to easily get connection information which is needed to setup these Microsoft Health Checks. 
```

> For example, to use the Steeltoe MySql connector but use Microsoft Health Checks make these changes to Startup.cs:

Reviewer: Should this explain that Steeltoe will use its own health check as a fall-back if the MSFT version is not found? This example does not explain why it works and the line of Steeltoe code looks the same either way
